docs/pages/web/popovereducational.js

```diff
@@ -0,0 +1,628 @@
+// @flow strict
+import { type Node } from 'react';
+import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
+import docgen, { type DocGen } from '../../docs-components/docgen.js';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable.js';
+import MainSection from '../../docs-components/MainSection.js';
+import Page from '../../docs-components/Page.js';
+import PageHeader from '../../docs-components/PageHeader.js';
+import QualityChecklist from '../../docs-components/QualityChecklist.js';
+
+export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader
+        name={generatedDocGen?.displayName}
+        description={generatedDocGen?.description}
+        defaultCode={`
```

Comment:
![Screen Shot 2023-02-27 at 2 21 41 PM](https://user-images.githubusercontent.com/12059539/221699330-7ee4e94a-f980-4811-898f-b179dff00955.png)

This seems a bit odd without a target element for it to be pointing to. Can we add a button or something here?

Reply:
Sure, I just wanted to highlight the component itself, as it's not a "paired component"

DONE